accord-core/src/main/java/accord/local/Node.java

```diff
@@ -894,6 +900,13 @@ public <R> CoordinationAdapter<R> coordinationAdapter(TxnId txnId, Kind kind)
         return coordinationAdapters.get(txnId, kind);
     }
 
+    public void updateMinHlc(long minHlc)
+    {
+        commandStores().forEach(safeStore -> {
```

Comment:
do we care that this won't affect new command stores that are added later?

Reply:
Looking into this now, but yes it is a problem if new ones are constructed without getting the min HLC from somewhere.